src/components/MovieDetail/MovieDetail.js

```diff
@@ -1,18 +1,17 @@
-import React, { useEffect } from "react";
-import "./MovieDetail.scss";
-import { useParams } from "react-router";
-import { useDispatch, useSelector } from "react-redux";
+import React, { useEffect } from 'react';
+import './MovieDetail.scss';
+import { useParams } from 'react-router';
+import { useDispatch, useSelector } from 'react-redux';
 import {
   fetchAsyncMovieOrShowDetail,
   getSelectedMovieOrShow,
   removeSelectedMovieOrShow,
-} from "../../features/movies/movieSlice";
+} from '../../redux/movies/movieSlice';
 
 const MovieDetail = () => {
```

Comment:
- [ ] Once more, kindly make sure the design of the detail page match the one provided in the design guidelines